buy_together_app/views.py

```diff
@@ -7,3 +9,34 @@ def main_page(request):
 
 def description_page(request):
     return render(request, 'buy_together_app/description.html')
+
+
+def signup_user(request, msg=""):
+    form = SupplierForm()
+    formm = ClientForm()
+    context = {'form': form, 'formm': formm, 'message': [msg]}
```

Comment:
Can we make it more descriptive?
```suggestion
    supplier_form = SupplierForm()
    client_form = ClientForm()
    context = {'supplier_form': supplier_form, 'client_form': client_form, 'message': [msg]}
```